app/frontend/hooks/resources/use-jurisdiction.ts

```diff
@@ -23,7 +23,8 @@ export const useJurisdiction = () => {
           setCurrentJurisdiction(jurisdictionId)
         }
       } catch (e) {
-        setError(e instanceof Error ? e : new Error(t("errors.fetchJurisdiction")))
+        console.error(e.message)
```

Comment:
prolly best to wrap console outputs to if in dev mode from environment variables

Reply:
lets not do this so that we can get more specific feedback from testers and users experiencing erros

Reply:
@bkmaibach We really shouldn't be exposing errors to console. Normally this would be handled by sentry. If it's just for testing and qa, create an error catcher function and put the console.error inside that. This will ensure that if and when we add error logging via Sentry or other method, we don't have to refactor the whole code base for console.errors